src/AdminCli/CliCommands/ListProductsCommand.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections.Generic;
+using AdminCli.HttpAccess;
+using McMaster.Extensions.CommandLineUtils;
+
+namespace AdminCli.CliCommands;
+
+public sealed class ListProductsCommand : ICliCommand
+{
+    public ListProductsCommand(IHttpService httpService)
+    {
+        HttpService = httpService;
+    }
+
+    private IHttpService HttpService { get; }
+
+
+    public void ConfigureCommand(CommandLineApplication config)
+    {
+        config.Description = "Lists all available products.";
+        config.OnExecuteAsync(async cancellationToken =>
+        {
+            var products = await HttpService.GetAsync<List<ProductListModel>>("/products", cancellationToken);
+            if (products is not { Count: >0 })
```

Comment:
I think `if (!products.Any())` or `if (products.Count == 0)` is way more readable.